doc/userguide/devguide/contributing/backports-guide.rst

```diff
@@ -0,0 +1,107 @@
+========================
+Suricata Backports Guide
+========================
+
+This document describes the processes used to backport content to previous
+Suricata releases. Most often, this means security and/or bug fixes;
+however, in some cases, features may be backported to previous Suricata releases.
+
+There are multiple versions of Suricata at any given time:
+    * Master
+    * Current release
+    * Release prior to current release
+
+For example, at the moment, there are 3 releases based on these Suricata branches:
+    * master: 8.0.0-dev, current development branch
+    * main-7.0.x: current stable release (note we're changing our naming conventions)
+    * master-6.0.x: previous stable release
+
+The next sections discuss when and what to backport, and some guidelines when
+doing so.
+
+What should be backported?
+--------------------------
+
+Usually, when the team creates a ticket, we'll add the *Needs backport* related
+labels, so necessary backporting tickets will be automatically created. If you
+are working on a ticket that doesn't have such labels, nor backporting tasks
+associated, it probably doesn't need backporting. But sometimes we'll miss those.
+
+There can be cases where backports may be "missed" -- some issues may not be
+labeled as needing backports and some PRs may be merged without an issue.
```

Comment:
Perhaps, these should go in an Exceptions or something section at the end? Thoughts?

Reply:
I'll add a note in this section, I think it's easier to keep context, like that. 🤔